### PR TITLE
Add tips for running rosbags on WSL

### PR DESCRIPTION
**Pull request by Reschivon**
## Brief Description

No code, just a README file.

## Related Issues

Closes ...

Opens ...

## Tasks

- [ ] Finished all sub-tasks (from issues)
- [ ] Updated Documentation
- [ ] Tests written/updated and implemented to CI

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,8 +105,8 @@ Design docs for the major components under construction.
 
 - [Autonomous Navigation Stack](https://docs.google.com/document/d/1H4zmVq_6n9eVdnM5NY5JfDTC1Pg-OLNYgg7p091wSyc/edit?usp=sharing)
 - [Race State System](https://docs.google.com/document/d/1WoQyybG2NdWSwledeUXwAQzwZzE1zPwBlswZRG5uL4k/edit?usp=sharing)
-
-Please follow ROS standard naming and project conventions:
+- [Running ROSbags on WSL](rosbag_tips.md)
+  Please follow ROS standard naming and project conventions:
```
Comment (alanssitis) [inline README.md:109]: They are two different bullet points

File: rosbag_tips.md
Comment (alanssitis) [inline rosbag_tips.md:1]: markdown file names should be capitalized. Perhaps `WSL_ROSBAG_TIPS.md`

**Review by alanssitis — CHANGES_REQUESTED** [review]

**Comment by Reschivon** [comment]
@alanssitis can you approve this

File: README.md [hunk 2]
```diff
@@ -105,8 +105,7 @@ Design docs for the major components under construction.
 
 - [Autonomous Navigation Stack](https://docs.google.com/document/d/1H4zmVq_6n9eVdnM5NY5JfDTC1Pg-OLNYgg7p091wSyc/edit?usp=sharing)
 - [Race State System](https://docs.google.com/document/d/1WoQyybG2NdWSwledeUXwAQzwZzE1zPwBlswZRG5uL4k/edit?usp=sharing)
-
-Please follow ROS standard naming and project conventions:
-
```
Comment (alanssitis) [inline README.md:110]: Woops I mischaracterized previous thing. It was a sub-header.

File: WSL_ROSBAG_TIPS.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+The bag file utility in ros2 `ros2 bag` has compatibility settings for running bags recorded with ROS1. The command is `ros2 bag play -s rosbag_v2 bagfile.bag`, but chances are you do not have the `rosbag_v2` plugin installed
+
+1. First, add the (noetic apt repository)[http://wiki.ros.org/noetic/Installation/Ubuntu], or that of another ROS1 version.
+
+`sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list'`
+
+`curl -s https://raw.githubusercontent.com/ros/rosdistro/master/ros.asc | sudo apt-key add -`
+
+`sudo apt update` 2. Then install the bag compatibility plugin `sudo apt install ros-<distro>-rosbag2-bag-v2-plugins`. This will install its dependency, `ros-distro-ros1-bridge`, which further requires some `noetic` packages 3. In order to have ROS 1 and 2 active together, we must source `/opt/ros/noetic/setup.bash` and `/opt/ros/foxy/setup.bash`, _in that order_. If you are sourcing foxy in your `.bashrc` already, add this before it:
```
Comment (alanssitis) [inline WSL_ROSBAG_TIPS.md:20]: Numbered entry 2 and 3 fix!

**Review by alanssitis — CHANGES_REQUESTED** [review]
Some minor changes.

**Pull request merged**